tests/unit/swatch.test.js

```diff
@@ -0,0 +1,41 @@
+import "@testing-library/jest-dom"
```

Comment:
I would really prefer to have all test files live with the components. I'd like to see this `.test.js` file sit in the component folder with it, rather than in a separate `tests` folder.